### PR TITLE
feat: mana test persistence

### PR DESCRIPTION
**Pull request by acha-bill**
Integration test for mana persistence.

Part of #800

### CODE REVIEW

File: tools/integration-tests/tester/tests/mana/mana_test.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import (
+	"testing"
+	"time"
+
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/tests"
+	"github.com/stretchr/testify/require"
+)
+
+func TestManaPersistence(t *testing.T) {
+	n, err := f.CreateNetwork("mana_TestPersistence", 1, 0, true)
```
Comment (lzpap) [inline tools/integration-tests/tester/tests/mana/mana_test.go:12]: let's create at least 5 nodes. If you take a look at the faucet tests, there is usually 1 out of 5 that fails the faucet persistence test for the same reason this test should fail

Comment (acha-bill) [reply tools/integration-tests/tester/tests/mana/mana_test.go:12]: yeah but that shouldn't matter here. This tests that `a` node persists its mana and it starts up with the stored values.
Fixing #749 is another thing.

Comment (lzpap) [reply tools/integration-tests/tester/tests/mana/mana_test.go:12]: okay

File: tools/integration-tests/tester/framework/parameters.go [hunk 2]
```diff
@@ -42,6 +42,8 @@ var (
 	ParaPoWFaucetDifficulty = 2
 	// ParaSyncBeaconOnEveryNode defines whether all nodes should be sync beacons.
 	ParaSyncBeaconOnEveryNode = false
+	// ParaManaOnEveryNode defines whether all nodes should have mana enabled.
+	ParaManaOnEveryNode = false
```
Comment (lzpap) [inline tools/integration-tests/tester/framework/parameters.go:46]:
```suggestion
	ParaManaOnEveryNode = true
```
mana is an essential component which is enabled by default on a node, why wouldn't we enable it then on all nodes in tests?

File: tools/integration-tests/tester/tests/mana/mana_test.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		err = peer.Start()
+		require.NoError(t, err)
+	}
+
+	// wait for peers to start
+	time.Sleep(5 * time.Second)
+
+	info, err = peers[0].Info()
+	require.NoError(t, err)
+	manaAfter := info.Mana
```
Comment (lzpap) [inline tools/integration-tests/tester/tests/mana/mana_test.go:44]: with default mana parameters, we know that during this test, `manaBefore` < `manaAfter` should hold true. maybe add an extra check for this?

**Review by lzpap — APPROVED** [review]

**Pull request merged**